coalib/bears/requirements/PythonRequirement.py

```diff
@@ -0,0 +1,19 @@
+from coalib.bears.requirements.PackageRequirement import PackageRequirement
+
+
+class PythonRequirement(PackageRequirement):
+    """
+    This class is a subclass of ``PackageRequirement``, and helps specifying
+    requirements from ``pip``, without using the manager name.
+    """
+
+    def __init__(self, package, version=""):
+        """
+        Constructs a new PythonRequirement, using the PackageRequirement
+        constructor.
+
```

Comment:
in fact `__init__` needs to be tested here, but there's a meaningful test you can use:
```
PythonRequirement("pack", version="1.2.3") == PackageRequirement("pip", "pack", version="1.2.3")
```